snippets/all_unique.md

```diff
@@ -0,0 +1,16 @@
+### all_unique
+
+Checks a flat list for all unique values. Returns True if list values are all unique and False if list values aren't all unique.
+
+This function compares the length of the list with length of the set() of the list. set() removes duplicate values from the list.
+
+``` python
+def all_unique(lst):
+    return len(lst) == len(set(lst))
+```
+
+``` python
+x = [1,2,3,4,5,6]
+all_unique(x) # result
```

Comment:
Replace `#result` with `True`

Reply:
Also, test file for one snippet is not generated. Please try running `test.py` again.

Reply:
I generated those files by hand following the schema of the others, I hadn't written any specific tests because I had not had time to look into pytape's syntax yet. 
I'll try running `test.py`

Reply:
Do not bother about the tests. `pytape` is created by me and is still under development. Just run test.py script